### PR TITLE
Remove 'ignored and undocumented' tee -u flag

### PR DESCRIPTION
**Pull request by msingle**
Signed-off-by: Matt Singletary <matt.singletary@gmail.com>

### CODE REVIEW

**Comment by bldybuild** [comment]
Commit dae65fab04e863f3607f46b3db1e0cf867efee4f is not signed-off.

Please fix these issues:

>	1. Dan Cross has not signed-off dae65fa.


If you'd like more information on how to sign your commits please visit [signed-off-by.me](https://signed-off-by.me)

File: sys/src/cmd/tee.c [hunk 1]
```diff
@@ -37,11 +37,6 @@ main(int argc, char **argv)
 		atnotify(intignore, 1);
 		break;
 
-	case 'u':
-		uflag++;
```
Comment (dancrossnyc) [inline sys/src/cmd/tee.c]: You'll probably want to remove the flag, too.

Comment (msingle) [inline sys/src/cmd/tee.c]: Thanks for the catch!

**Comment by bldybuild** [comment]
Commit 7252f594cc69f152a88eaba629b3eaa2f8c28421 is not signed-off.

Please fix these issues:

>	1. Matt Singletary has not signed-off 7252f59.


If you'd like more information on how to sign your commits please visit [signed-off-by.me](https://signed-off-by.me)

**Comment by msingle** [comment]
double of commit because I was trying to make github stop warning me that this branch is out-of-date with base

**Comment by rminnich** [comment]
hey matt, sorry, can you do a rebase and push again ...

**Comment by bldybuild** [comment]
Commit e1f8b47cbed30325ce96df949786cf3e09564d17 is not signed-off.

Please fix these issues:

>	1. ron minnich has not signed-off e1f8b47.


If you'd like more information on how to sign your commits please visit [signed-off-by.me](https://signed-off-by.me)

**Pull request merged**